src/expr/src/scalar/func.rs

```diff
@@ -3027,6 +3028,18 @@ fn timezone_interval_timestamptz(a: Datum<'_>, b: Datum<'_>) -> Result<Datum<'st
     }
 }
 
+#[sqlfunc(
+    output_type_expr = "ScalarType::Record {
+                fields: [
+                    (\"abbrev\".into(), ScalarType::String.nullable(false)),
+                    (\"base_utc_offset\".into(), ScalarType::Interval.nullable(false)),
+                    (\"dst_offset\".into(), ScalarType::Interval.nullable(false)),
```

Comment:
You could use raw string literals (`r#" ... "#`) instead, so that you don't have to escape the `"`s inside the literal.

Reply:
Yup, that works! Thanks for the suggestion.